lib/widgets/directions_drawer.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:concordia_navigation/providers/map_data.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class DirectionsDrawer extends StatefulWidget {
+  @override
+  _DirectionsDrawerState createState() => _DirectionsDrawerState();
+}
+
+class _DirectionsDrawerState extends State<DirectionsDrawer> {
+  bool isCollapsed = false;
```

Comment:
I think you forgot to remove this. Might as well make this a stateless widget.

Reply:
Yes, you're right, going to convert it to Stateless Widget.